virttest/virt_vm.py

```diff
@@ -1408,6 +1408,30 @@ def get_cpu_count(self, check_cmd='cpu_chk_cmd', connect_uri=None):
         out = self.session.cmd_output_safe(cmd)
         return int(re.search("\d+", out, re.M).group())
 
+    @session_handler
+    def check_cpu_topology(self, check_cmd="cpu_topo_chk_cmd"):
+        """
+        check the cpu topology of the VM.
+        """
+        cmd = self.params.get(check_cmd)
+        out = self.session.cmd_output_safe(cmd)
+        os_type = self.params.get('os_type')
+        if os_type == "linux":
```

Comment:
Should we also check dies too, from /sys/devices/system/cpu/cpuX/topology/

Reply:
It is only available on a specific platform, I don't think we need to check it.